content/foundations/color.mdx

```diff
@@ -2,65 +2,191 @@
-| `canvas-inset`     | Surface on views where the content is the primary focus. For example, feeds and timelines.     |
-| `border-default`     | Main border to create boundaries. Critical to understanding the interface's hierarchy.    |
-| `border-muted`     | Secondary border to emphasize separation between elements.     |
-| `border-subtle`    | Low opacity border designed to blend with backgrounds like images on avatars or primary buttons.     |
-| `fg-default`    | Provides high readability. Default text and icons, and most foregrounds.      |
-| `fg-muted`    | Provides medium readability. Used to create hierarchy in combination with `fg-default`.      |
-| `fg-subtle`    | Provides low contrast. Used for placeholder text.      |
-| `fg-on-emphasis`    | Used for text and icons on color backgrounds.     |
-
-## Color roles 
-Color roles are defined by the meaning they convey in the UI. Primer has 8 color roles with 4 variants each. 
-
-![Color role anatomy: color role prefix + variant suffix](https://user-images.githubusercontent.com/6951037/126636938-8674c80c-c9cb-425c-b6d5-d1ddcc742fcb.png)
-
-### Global roles
-
-| Prefix | Role |
-| -------- | -------- |
-| `neutral-` | Use to highlight content without any added meaning. |
-| `accent-` | Use to draw attention to interactive elements. |
-| `success-` | Use to expresses the completion or positive outcome of a task. |
-| `attention-` | Use to warn of pending tasks or highlight active content.  |
-| `severe-` | Use when there are more than 3 levels of states, for example in heatmaps. |
-| `danger-` | Use to inform of error or another negative message. |
-
-### GitHub-specific roles
-
-| Prefix | Role |
-| -------- | -------- |
-| `done-` | Completion color for productivity and code review workflows. |
-| `sponsors-` | Color suite for all Sponsors-related content. |
-
-### Variants
-
-| Suffix | Role |
-| -------- | -------- |
-| `-fg` | Text and icons. |
-| `-emphasis` | High contrast for background and borders. |
-| `-emphasis-plus` | Highest contrast for background and borders. Use only with the `neutral-` prefix. |
-| `-muted` | Medium contrast alpha value, recommended for borders. |
-|`-subtle` | Low contrast solid color, recommended for backgrounds. |
+## How to use color for product UI
+Primer delivers colors in the form of [variables](https://primer.style/primitives/colors) or design tokens. Color variables can be grouped into different groups based on their purpose:
+
+- **Functional:** To convey a meaning or a state. For example, from the functional perspective the color green is used to reinforce positive messaging. In a functional system green variables are named with the suffix `.success`
+- **Presentational:** To represent a color. For example, the color steps in the Primer scale are named by color and lightness. Names are such as `scale.blue.5`. These variables don't currently support color modes. There is no available system to use colors in this form yet.
```

Comment:
> There is no available system to use colors in this form yet.

Need to rephrase this: What is this trying to say? What is the guidance here, what do we want people to do or not do?